chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -47,13 +47,14 @@
 
   bash "disable_openstack_services" do
     code <<-EOF
-      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1) \
+      for i in $(systemctl list-units openstack* --no-legend | cut -d" " -f1 | grep -v keystone | grep -v neutron) \
                drbd.service \
                pacemaker.service;
       do
         systemctl disable $i
       done
     EOF
+    not_if { ::File.exist?("/usr/sbin/crm") }
```

Comment:
I'm actually not sure, but I think pacemaker should not be happy by something external disabling services.

I could revert this later on, but would like to try it this way.